test/integration_test.rb

```diff
@@ -23,6 +23,12 @@ def test_no_test_is_selected
         assert_equal "", output[:stdout]
       end
 
+      def test_all_tests_are_selected_without_mapping
+        git_rm_and_commit("#{@repo.workdir}/.ttnt", 'Remove .ttnt')
+        output = rake('ttnt:test:run')
+        assert_match '2 runs, 6 assertions, 0 failures', output[:stdout]
```

Comment:
Wow, this is quite clever! Thank you :smile: 
Fixed in: https://github.com/Genki-S/ttnt/commit/c7d316d8c042d35e7579406b4bad55d80b22ee22